uint32 mobileye_can_message_number
uint32 msg_count
uint32 msg_status
float64 tosec
uint32 time
uint32 Con_Left
uint32 LDW_Left
uint32 Type_Left
float64 Dis_Left
uint32 Con_Right
uint32 LDW_Right
uint32 Type_Right
float64 Dis_Right
uint32 Head_Valid
float64 Head_Mea
uint32 LDW_OFF
uint32 LDW_Left_On
uint32 LDW_Right_On
uint32 FCW_On
uint32 Left_Crossing_Event
uint32 Right_Crossing_Event
uint32 P_FCW
uint32 P_DZ
uint32 TSR_En
uint32 TSR_W_Lv
uint32 Head_W_Lv
uint32 HW_R_En
uint32 Sign1
uint32 Sign2
uint32 Sign3
uint32 Sign4
uint32 S_Sign1
uint32 S_Sign2
uint32 S_Sign3
uint32 S_Sign4
uint32 HLB_dec
uint32 L_Beam
float64 L_Cur
float64 L_Head
uint32 Con_Area
uint32 R_LDW
uint32 L_LDW
float64 Yaw
float64 Pitch
uint32 Num_O
uint32 Timestamp
uint32 L_Close
uint32 R_Close
uint32 GO
uint32 Close_Car
uint32 L_Signal
uint32 R_Signal
uint32 Wiper
uint32 Lo_Beam
uint32 Hi_Beam
uint32 Speed
uint32 Sign_Type
uint32 S_Sign_Type
float64 Sign_X
float64 Sign_Y
float64 Sign_Z
uint32 Filter_Type

